src/server/pps/server/worker_rc.go

```diff
@@ -153,11 +177,11 @@ func (kd *kubeDriver) workerPodSpec(options *workerOptions, pipelineInfo *pps.Pi
 		Name:  client.PPSPipelineNameEnv,
 		Value: pipelineInfo.Pipeline.Name,
 	}, {
-		Name:  "LOKI_SERVICE_HOST_VAR",
-		Value: kd.config.LokiHostVar,
+		Name:  "LOKI_SERVICE_HOST",
```

Comment:
Why does the worker need to know about loki?

Reply:
My initial thought was that it is so `get logs -p <pipeline name>` works. But I guess that it doesn't matter because its pachd that is making the call out to loki and filtering for the pipeline. Yeah .. so not sure why these variables are needed unless the sidecar also needs to talk to loki.

Reply:
Logs are sent to loki by promtail reading the container logs from disk.  Logs are read from loki over its http API via the GetLogsLoki call in pachd.  So it's not obvious to me why this was ever passed thru.

Reply:
Also I deduped the `LOKI_SERVICE_HOST` and `LOKI_SERVICE_PORT` in the manifest